feature-libs/product/configurators/rulebased/cpq/cpq.models.ts

```diff
@@ -83,6 +79,18 @@ export namespace Cpq {
     quantity?: string;
   }
 
+  /**
+   *
+   * An interface representing the structure for update of CPQ configuration attribute.
+   */
+  export interface UpdateAttribute {
+    configurationId: string;
+    stdandardAttrCode: string;
+    attributeValueIds?: string;
```

Comment:
Why not simply **valueIds**?

Reply:
This is the name of the CPQ API parameter